client/wfnews-war/src/main/angular/src/app/utils/index.ts

```diff
@@ -789,3 +789,51 @@ export function showPanel(panelClass: string) {
 
 
 export const isAndroidViaNavigator = () => navigator.platform.includes('Linux') || navigator.platform.includes('Android');
+
+export function displayItemTitle(identifyItem) {
```

Comment:
same with draggle-panel.component, let it import from here please

Reply:
Done